bpf/lib/conntrack.h

```diff
@@ -327,10 +327,10 @@ __ct_lookup(const void *map, struct __ctx_buff *ctx, const void *tuple,
 			}
 
 			*monitor = TRACE_PAYLOAD_LEN;
-			if (ct_entry_alive(entry))
-				break;
-			__ct_update_timeout(entry, bpf_sec_to_mono(CT_CLOSE_TIMEOUT),
-					    dir, seen_flags, CT_REPORT_FLAGS);
+
+			lifetime = ct_entry_alive(entry) ? CT_CLOSING_TIMEOUT : CT_CLOSE_TIMEOUT;
```

Comment:
Looks like you still need to define the `__u32 lifetime` :).

Reply:
Thanks for catching it. I've add the type.